config/webpack.vendor.js

```diff
@@ -43,12 +43,9 @@ module.exports = [
    * Brace related
    */
   'brace',
-  'brace/mode/javascript',
   'brace/mode/ruby',
-  'brace/mode/html_ruby',
   'brace/mode/json',
   'brace/mode/sh',
-  'brace/mode/xml',
```

Comment:
xml may be useful, some templates use it (for example, AutoYaST SLES default)

Reply:
that was the original reason I've added it, but i'm not sure its worth extra 300k for it..?